docs/pages/release_notes.md

```diff
@@ -1,5 +1,63 @@
 Release Notes {#releasenotes}
 ==============
+v3.4.0
+==============
+The source code with submodules can be downloaded directly from the following link:
+http://arrayfire.com/arrayfire_source/arrayfire-full-3.4.0.tar.bz2
+
+Features
+----------
+* New [interpolation methods](https://github.com/arrayfire/arrayfire/issues/1562) for \ref af::resize(), \ref af::transform(), \ref af::approx1() and \ref af::approx2()
+* Support for complex mathematical functions
+* New Forge graphics [integration](https://github.com/arrayfire/arrayfire/pull/1555)!
+    * Vector Field plotting functionality
+    * API updates
+* \ref af::medfilt1(): [Median filter for 1-d signals](https://github.com/arrayfire/arrayfire/pull/1479)
+* af::RandomEngine(): New [random number generators](https://github.com/arrayfire/arrayfire/issues/868)//TODO
+    * Philox
+    * Threefry
+    * Mersenne Twister
+* \ref af::sparse(): [Sparse matrix support for all backends](https://github.com/arrayfire/arrayfire/issues/821)
+* \ref af::scan(): New [generalized scan](https://github.com/arrayfire/arrayfire/issues/388) functions
+* \ref af::moments(): New [image moments](https://github.com/arrayfire/arrayfire/pull/1453) functions
```

Comment:
Use `[Image Moments](\ref moments_mat)`
Similar to sparse.
That way the entire group of functions can be linked.